music_assistant/server/providers/deezer/gw_client.py

```diff
@@ -150,4 +150,54 @@ async def get_deezer_track_urls(self, track_id):
         if error := result_json["data"][0].get("errors"):
             raise DeezerGWError("Received an error from API", error)
 
+        print(await self.mark_track_played(track_id=track_id))
```

Comment:
please either drop the non working code or finish it

Reply:
Yeah I was not planning to already do a PR but then the deezer-python version came out and and I wasn't home so I can't fix it yet.